packages/solid-query/src/createQuery.ts

```diff
@@ -86,6 +109,19 @@ export function createQuery<
     'queryKey'
   >,
 ): CreateQueryResult<TData, TError>
+export function createQuery<
+  TQueryFnData = unknown,
+  TError = unknown,
+  TData = TQueryFnData,
+  TQueryKey extends SolidQueryKey = SolidQueryKey,
+>(
+  queryKey: TQueryKey,
+  queryFn: QueryFunction<TQueryFnData, ReturnType<TQueryKey>>,
+  options: Omit<
```

Comment:
`options` needs to be defined for this overload, otherwise, typescript will require `options` to be at least an empty object to fallback to the following overloads